actions/hosts.last_checkin.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: hosts.last_checkin
+pack: foreman
+description: List all hosts that haven't reported back 
+runner_type: orquesta
+entry_point: workflows/hosts.last_checkin.yaml
+enabled: true
+parameters:
+  server:
+    required: true
+    type: string
+  username: 
+    required: true
+    type: string
+  password:
+    required: true
+    type: string
+  search:
```

Comment:
instead of taking a generic "search" let's make this a a "time"  that can either be a timestamp or one of the text times like "3 days ago"